src/domains/wallet-app/api/payment-wallet/v1/_get_wallets_by_user_and_walletId.xml.tpl

```diff
@@ -0,0 +1,99 @@
+        <set-header name="Authorization" exists-action="override">
+          <value>@("Bearer " + (string)context.Variables.GetValueOrDefault("walletToken"))</value>
+        </set-header>
+    </send-request>
+    <choose>
+      <when condition="@(((IResponse)context.Variables["user-auth-body"]).StatusCode != 200)">
+        <return-response>
+          <set-status code="502" reason="Bad Gateway" />
+          <set-header name="Content-Type" exists-action="override">
+              <value>application/json</value>
+          </set-header>
+          <set-body>
+              {
+              "title": "Error get user",
+              "status": 502,
+              "detail": "There was an error getting user info"
+              }
+          </set-body>
+        </return-response>
+      </when>
+    </choose>
+    <set-variable name="userAuth" value="@(((IResponse)context.Variables["user-auth-body"]).Body.As<JObject>())" />
+    <!-- Get User IO END-->
+    <!-- Post Token PDV START-->
+    <send-request ignore-error="true" timeout="10" response-variable-name="pdv-token" mode="new">
+      <set-url>${pdv_api_base_path}/tokens</set-url>
+      <set-method>PUT</set-method>
+      <set-header name="x-api-key" exists-action="override">
+          <value>{{personal-data-vault-api-key}}</value>
+      </set-header>
+      <set-body>@{
+        JObject requestBody = (JObject)context.Variables["userAuth"];
+        return new JObject(
+                new JProperty("pii",  (string)requestBody["fiscal_code"])
+            ).ToString();
+          }</set-body>
+    </send-request>
+    <choose>
+      <when condition="@(((IResponse)context.Variables["pdv-token"]).StatusCode != 200)">
+        <return-response>
+          <set-status code="502" reason="Bad Gateway" />
+          <set-header name="Content-Type" exists-action="override">
+              <value>application/json</value>
+          </set-header>
+          <set-body>
+              {
+              "title": "Error PDV",
+              "status": 502,
+              "detail": "There was an error during PDV encoding"
+              }
+          </set-body>
+        </return-response>
+      </when>
+    </choose>
+    <set-variable name="pdvToken" value="@(((IResponse)context.Variables["pdv-token"]).Body.As<JObject>())" />
+    <set-variable name="fiscalCodeTokenized" value="@((string)((JObject)context.Variables["pdvToken"])["token"])" />
+    <choose>
+      <when condition="@(String.IsNullOrEmpty((string)context.Variables["fiscalCodeTokenized"]))">
+          <return-response>
+              <set-status code="502" />
+              <set-header name="Content-Type" exists-action="override">
+                  <value>application/json</value>
+              </set-header>
+              <set-body>@{
+          {
+                "title": "Bad gateway - Invalid PDV response",
+                "status": 502,
+                "detail": "Cannot tokenize fiscal code"
+          }
+        }</set-body>
```

Comment:
```suggestion
              <set-body>
                {
                      "title": "Bad gateway - Invalid PDV response",
                      "status": 502,
                      "detail": "Cannot tokenize fiscal code"
                }
             </set-body>
```
This should be a syntax error, right? please remove @{} syntax since no code will be executed into this tag